webrecorder/templates/collection_info.html

```diff
@@ -26,6 +26,8 @@
 <script>
     var can_admin = {{ 'true' if can_admin() else 'false' }};
     var can_write = {{ 'true' if can_write() else 'false' }};
+
+    var init_selected_recs = {{ rec_list }};
```

Comment:
That results in duplicating selection logic server side and client side. In general, I think if something is going to be changing dynamically client side, much better to also set initial state client side, rather than doing same thing in two different places. In this case, the one-line JS inject is worth the tradeoff.